[2025-04-23T15:49:05.798+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-23T15:49:05.813+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_data_etl_dbt.dbt_snapshot manual__2025-04-23T15:16:33.263567+00:00 [queued]>
[2025-04-23T15:49:05.939+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_data_etl_dbt.dbt_snapshot manual__2025-04-23T15:16:33.263567+00:00 [queued]>
[2025-04-23T15:49:05.941+0000] {taskinstance.py:2865} INFO - Starting attempt 3 of 3
[2025-04-23T15:49:05.997+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dbt_snapshot> on 2025-04-23 15:16:33.263567+00:00
[2025-04-23T15:49:06.027+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=30894) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-23T15:49:06.034+0000] {standard_task_runner.py:72} INFO - Started process 30900 to run task
[2025-04-23T15:49:06.033+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stock_data_etl_dbt', 'dbt_snapshot', 'manual__2025-04-23T15:16:33.263567+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/Stock_built_ELT.py', '--cfg-path', '/tmp/tmpji8vnxf_']
[2025-04-23T15:49:06.040+0000] {standard_task_runner.py:105} INFO - Job 73: Subtask dbt_snapshot
[2025-04-23T15:49:06.163+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_data_etl_dbt.dbt_snapshot manual__2025-04-23T15:16:33.263567+00:00 [running]> on host 712e28762a67
[2025-04-23T15:49:06.344+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_data_etl_dbt' AIRFLOW_CTX_TASK_ID='dbt_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T15:16:33.263567+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T15:16:33.263567+00:00'
[2025-04-23T15:49:06.348+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-23T15:49:06.356+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-23T15:49:06.358+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt snapshot --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2025-04-23T15:49:06.375+0000] {subprocess.py:86} INFO - Output:
[2025-04-23T15:49:10.339+0000] {subprocess.py:93} INFO - [0m15:49:10  Running with dbt=1.7.19
[2025-04-23T15:49:11.985+0000] {subprocess.py:93} INFO - [0m15:49:11  Registered adapter: snowflake=1.7.1
[2025-04-23T15:49:12.050+0000] {subprocess.py:93} INFO - [0m15:49:12  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-04-23T15:49:12.051+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-04-23T15:49:12.056+0000] {subprocess.py:93} INFO - - models.stock_data_transform_dbt.marts
[2025-04-23T15:49:12.101+0000] {subprocess.py:93} INFO - [0m15:49:12  Found 4 models, 1 snapshot, 1 source, 0 exposures, 0 metrics, 430 macros, 0 groups, 0 semantic models
[2025-04-23T15:49:12.105+0000] {subprocess.py:93} INFO - [0m15:49:12
[2025-04-23T15:49:15.866+0000] {subprocess.py:93} INFO - [0m15:49:15  Concurrency: 1 threads (target='dev')
[2025-04-23T15:49:15.874+0000] {subprocess.py:93} INFO - [0m15:49:15
[2025-04-23T15:49:15.887+0000] {subprocess.py:93} INFO - [0m15:49:15  1 of 1 START snapshot snapshots.stock_data_snapshot ............................ [RUN]
[2025-04-23T15:49:19.347+0000] {subprocess.py:93} INFO - [0m15:49:19  1 of 1 OK snapshotted snapshots.stock_data_snapshot ............................ [[32mSUCCESS 0[0m in 3.45s]
[2025-04-23T15:49:19.358+0000] {subprocess.py:93} INFO - [0m15:49:19
[2025-04-23T15:49:19.360+0000] {subprocess.py:93} INFO - [0m15:49:19  Finished running 1 snapshot in 0 hours 0 minutes and 7.25 seconds (7.25s).
[2025-04-23T15:49:19.387+0000] {subprocess.py:93} INFO - [0m15:49:19
[2025-04-23T15:49:19.389+0000] {subprocess.py:93} INFO - [0m15:49:19  [32mCompleted successfully[0m
[2025-04-23T15:49:19.389+0000] {subprocess.py:93} INFO - [0m15:49:19
[2025-04-23T15:49:19.389+0000] {subprocess.py:93} INFO - [0m15:49:19  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-04-23T15:49:20.276+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-23T15:49:20.337+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-23T15:49:20.338+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_data_etl_dbt, task_id=dbt_snapshot, run_id=manual__2025-04-23T15:16:33.263567+00:00, execution_date=20250423T151633, start_date=20250423T154905, end_date=20250423T154920
[2025-04-23T15:49:20.408+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-23T15:49:20.454+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-23T15:49:20.456+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
